instagram-app/src/components/CommentSection/Comment.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from '../SearchBar/SearchBarStyle';
+
+class Comment extends React.Component {
+  constructor() {
+    super();
+  }
+
+  render() {
+    console.log(this.props.id);
```

Comment:
Try and remove `console.logs` to make your code neater before you commit a feature. 👍